scripts/pipeline/create_all_mlss.pl

```diff
@@ -594,43 +595,74 @@ sub make_named_species_set_from_XML_node {
 my $current_version = software_version();
 my %methods_not_worth_reporting = map {$_ => 1} qw(SYNTENY ENSEMBL_ORTHOLOGUES ENSEMBL_PARALOGUES ENSEMBL_HOMOEOLOGUES ENSEMBL_PROJECTIONS CACTUS_HAL_PW GERP_CONSTRAINED_ELEMENT GERP_CONSERVATION_SCORE);
 
+sub mlss2hash {
+    my $self = shift;
+    my $res = {
+        db_id            => $self->dbID,
+        name             => ($self->name ? $self->name : '(unnamed)'),
+        method_type      => $self->method->type,
+        species_set_name => $self->species_set->name,
+        species_set_id   => $self->species_set->dbID
+    };
+    if ($self->{'url'}) {
+        $res->{url} = $self->{'url'};
```

Comment:
I'd suggest to use hash key quotes consistently here; maybe by quoting everything...
```suggestion
    if ($self->{'url'}) {
        $res->{'url'} = $self->{'url'};
```
...or perhaps by quoting nothing:
```suggestion
    if ($self->{url}) {
        $res->{url} = $self->{url};
```
You could even sidestep things a little by fetching the original URL:
```suggestion
my $url = $self->get_original_url();
$res->{url} = $url if $url;
```